tests/smtp-long-DATA-line-02-frames/test.yaml

```diff
@@ -0,0 +1,23 @@
+requires:
+  min-version: 8
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 1
+    match:
+      alert.signature_id: 1
+- filter:
+    count: 1
+    match:
+      alert.signature_id: 2
```

Comment:
We should also test app-layer metadata logging

Reply:
adding